universal-application-tool-0.0.1/app/views/dev/AwsStorageDevViewStrategy.java

```diff
@@ -58,4 +68,21 @@ public ContainerTag getFileUploadForm(
         .withMethod("post")
         .withAction(request.actionLink());
   }
+
+  @Override
+  public ContainerTag renderFiles(ImmutableList<StoredFile> files, StorageClient client) {
```

Comment:
question (blocking): could the `StorageClient` be injected in the constructor for this class rather passed in here as a method param?

Reply:
Yup, revised so the client is injected